### DISCUSSION TITLE
🐞 - UI Issue (Z-index)

### DISCUSSION THREAD

**Discussion by Avivhdr** [community/community#19183] [Pull Requests] [answered]
OS: Big Sur 11.6.5
Firefox: 101.0.1 (64-bit)

![image](https://user-images.githubusercontent.com/23265867/175054216-dcb4d95d-2099-46af-8c49-7bce4ecb5bcf.png)

**Comment by melvincwng** [comment, marked as answer]
@Avivhdr, kindly report bugs to [Github Bug Report](https://support.github.com/contact/bug-report).

Thank you 😊

**Reply by Avivhdr** [reply]
https://support.github.com/ticket/personal/0/1678688
